src/scripts/main.js

```diff
@@ -1,3 +1,14 @@
 'use strict';
 
-// write your code here
+const rows = [...document.querySelectorAll('tr')];
+const column5 = rows.map((row) => {
+  return row.children[4];
+});
```

Comment:
The `column5` array is storing references to the original elements in the DOM. When you append these elements to another parent, they will be removed from their original location. Consider cloning these nodes if you need to maintain their original position.